policy/modules/contrib/spamassassin.te

```diff
@@ -659,12 +660,29 @@ optional_policy(`
 	gpg_manage_home_content(spamd_update_t)
 ')
 
+optional_policy(`
+	init_status(spamd_update_t)
+')
+
+optional_policy(`
+	mta_read_config(spamd_update_t)
+')
+
+optional_policy(`
+	spamassassin_systemctl(spamd_update_t)
```

Comment:
Interfaces like this should be used only outside the module. I believe something like `spamd_unit_file_t:file read_file_perms` would work unless you have more denials.

Reply:
SELinux denied spamd_update_t to start and status /usr/lib/systemd/system/spamassassin.service